src/components/RestaurantCard/RestaurantCard.tsx

```diff
@@ -1,35 +1,42 @@
-import "./RestaurantCard.scss";
+import styles from "./RestaurantCard.module.scss";
 import { Restaurant } from "../../types/types";
 import useGetScreenWidth from "../../hooks/useGetWidthScreen";
 import { UIConstants } from "../../shared/constants";
 interface RestaurantCardProps {
   restaurant: Restaurant;
+  className?: string;
 }
 
-const RestaurantCard = ({ restaurant }: RestaurantCardProps) => {
+const RestaurantCard = ({ restaurant, className }: RestaurantCardProps) => {
   const screenWidth = useGetScreenWidth();
   const isTablet = screenWidth < UIConstants.sizes.tabletWidth;
   const ratingImageSrcString: string = `/src/assets/images/ratings/rating${restaurant.rating}.svg`;
```

Comment:
This is a bad way to import something. This absolute path creates an opening for serious bugs. Can you see why?

Reply:
@nitzanpap  done